packages/dx-react-chart-bootstrap4/src/templates/axis/tick.jsx

```diff
@@ -1,33 +1,4 @@
-import * as React from 'react';
-import * as PropTypes from 'prop-types';
-import classNames from 'classnames';
+import { Axis } from '@devexpress/dx-react-chart';
+import { withClassName } from '../utils';
 
-export class Tick extends React.PureComponent {
-  render() {
-    const {
-      x1, x2, y1, y2, className, ...restProps
-    } = this.props;
-    return (
-      <path
-        d={`M ${x1} ${y1} L ${x2} ${y2}`}
-        className={classNames(
-          'dx-c-bs4-stroke-current-color dx-c-bs4-crisp-edges dx-c-bs4-axis-opacity',
-          className,
-        )}
-        {...restProps}
-      />
-    );
-  }
-}
-
-Tick.propTypes = {
-  x1: PropTypes.number.isRequired,
-  x2: PropTypes.number.isRequired,
-  y1: PropTypes.number.isRequired,
-  y2: PropTypes.number.isRequired,
-  className: PropTypes.string,
-};
-
-Tick.defaultProps = {
-  className: undefined,
-};
+export const Tick = withClassName('dx-c-bs4-stroke-current-color dx-c-bs4-crisp-edges dx-c-bs4-axis-opacity')(Axis.Tick);
```

Comment:
Can we support `array` syntax here to avoid too long strings? I mean:

```
withClassName([
'a',
'b',
'c',
...
])
```

Reply:
Yes, we can. Since there are cases when single string is provided, it seems that it would be better to support arbitrary amount of arguments (like it does underlying [classnames](https://www.npmjs.com/package/classnames)).

Reply:
Updated.